### PR TITLE
Update rubocop

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
I'm not particularly fond of the following cops:
- [Style/RescueModifier](https://github.com/theforeman/smart_proxy_dynflow/pull/57/commits/0c5693b3a00cbb2dfce2c4de280a2b7c951ea40d)
- [Style/Alias](https://github.com/theforeman/smart_proxy_dynflow/pull/57/commits/77f7ebba5d3688439b3f28a2255ebcecb2ccaf50)

I don't feel too strongly about the rest

**Comment by iNecas** [comment]
+1 to ditch those.

File: lib/smart_proxy_dynflow_core/callback.rb [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
-      def run
-        Request.send_to_foreman_tasks(input[:callback], input[:data])
+        private
+
+        def prepare_payload(callback, data)
+          { :callback => callback, :data => data }.to_json
+        end
+
+        # rubocop:disable Metrics/PerceivedComplexity
+        def ssl_options
```
Comment (iNecas) [inline lib/smart_proxy_dynflow_core/callback.rb]: Seems like `ssl_option` actually shouldn't be private, as it's being used here https://github.com/theforeman/smart_proxy_dynflow/pull/57/files#diff-3a71619fb07bb1f56160a122b3b42900R62 as `self.class.ssl_options`

**Comment by iNecas** [comment]
[test smart_proxy_dynflow]

**Pull request merged**